src/ray/raylet/node_manager.cc

```diff
@@ -702,6 +719,50 @@ void NodeManager::ProcessRegisterClientRequestMessage(
   }
 }
 
+void NodeManager::HandleDisconnectedActor(const ActorID &actor_id, bool was_local,
+                                          bool intentional_disconnect) {
+  auto actor_entry = actor_registry_.find(actor_id);
+  RAY_CHECK(actor_entry != actor_registry_.end());
+  auto &actor_registration = actor_entry->second;
+  RAY_LOG(INFO) << "The actor with ID " << actor_id << " died "
```

Comment:
Yep, actor dying is expected, but should be infrequent. I think this message is worth noticing, because if it become massive, we can know something is wrong. I prefer keeping `INFO`, what do you think?

Reply:
Hmm about how we only log if `!intentional_disconnect`?

Reply:
I changed it to debug. that's fine as well.